cmd/podman/secrets/create.go

```diff
@@ -0,0 +1,76 @@
+package secrets
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:   "create SECRET",
```

Comment:
Wrong use line. Should be `create SECRET FILE`

Reply:
FILE or `-` actually